config/schema/field_definitions.json

```diff
@@ -399,5 +399,21 @@
   "important_to_policy": {
     "description": "A flag set by editors to mark a document as being important to policy",
     "type": "boolean"
+  },
+
+  "country": {
+    "type": "identifier"
```

Comment:
Could you add a "description" field for each of these fields, indicating the type of content expected in it, and any special considerations.  This is our way of commenting the file, so that we have some idea what the existing fields are used for.

The comments should explain any special terminology used, too (eg, "promulgation").